setup.py

```diff
@@ -0,0 +1,25 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+setuptools.setup(
+    name="pangeobench",
+    scripts=["pangeobench"],
+    author="Haiying Xu",
+    author_email="haiyingx@ucar.edu",
+    description="Pangeo zarr benchmarking package",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    use_scm_version={
+        "version_scheme": "guess-next-dev",
+        "local_scheme": "dirty-tag",
+        "write_to": "benchmarks/version.py",
```

Comment:
How about removing this key `write_to`, and adding `benchmarking/__init__.py` with the following content (for retrieving version at runtime)

```python
from pkg_resources import DistributionNotFound, get_distribution
try:
    __version__ = get_distribution(__name__).version
except DistributionNotFound:
    # package is not installed
    __version__ = '0.0.0'
```

Reply:
This is good for users. But how can I get the version number during development?

Reply:
>  But how can I get the version number during development?

The changes I suggested will take care of this for us. We will need to update the command utility with a `--version` flag to allow something along the lines of 

```bash
pangeobench --version
```

Reply:
I plan to use bumpversion to control version, it can use one command to bump version's major/minor/patch up one. And it will be used for git tag or git commit afterwards. What do you think?

Reply:
It's my understanding that `bumpversion` and `setuptools_scm` work differently. 


|                                        | setuptools_scm                                                   | bumpversion                                                                   |
|----------------------------------------|------------------------------------------------------------------|-------------------------------------------------------------------------------|
| Single source of truth for the version | version control system tags                                      | `setup.py`                                                                      |
| Version propagation event              | script commit with changed version variable creates new VCS tags | tag creation triggers new build with updated version variable in `setup.py` script |

So, it seems to me that you want either option but not both options and the current setup.py is using `setuptools_scm` via `use_scm_version`.

Reply:
I thought bumpversion is better, since bumpversion can use during development, but scm only work on distribution.

Reply:
After discussed with Kevin, I switched back to setuptools_scm. Please have a look @andersy005